src/server/services/wmts/qgswmtsutils.cpp

```diff
@@ -34,15 +34,12 @@ namespace QgsWmts
 {
   namespace
   {
-    QMap< QgsUnitTypes::DistanceUnit, double> populateInchesPerUnit();
     QMap< QString, tileMatrixInfo> populateTileMatrixInfoMap();
 
     QgsCoordinateReferenceSystem wgs84 = QgsCoordinateReferenceSystem::fromOgcWmsCrs( GEO_EPSG_CRS_AUTHID );
 
-    double METERS_PER_INCH = 0.02540005080010160020;
-    QMap< QgsUnitTypes::DistanceUnit, double> INCHES_PER_UNIT = populateInchesPerUnit();
-    int tileWidth = 256;
-    int tileHeight = 256;
+    int tileSize = 256;
+    double POINTS_TO_M = 2.83464567 / 10000.0;
```

Comment:
Why `POINTS_TO_M` instead of `points_to_m`?

Reply:
Should it be `constexpr` ? (or at least `const`)